test/validation/api/scheduler/scheduler.c

```diff
@@ -1712,8 +1827,9 @@ static int create_queues(test_globals_t *globals)
 	num_sched = (prios * queues_per_prio * sched_types) + CHAOS_NUM_QUEUES;
 	num_plain = (prios * queues_per_prio);
 	while ((num_sched > default_config.num_queues ||
-		num_plain > capa.plain.max_num ||
-		num_sched + num_plain > capa.max_queues) && queues_per_prio) {
+		num_plain > queue_capa.plain.max_num ||
+		num_sched + num_plain > queue_capa.max_queues) &&
+		queues_per_prio) {
```

Comment:
Unless I'm mistaken queues_per_prio should be 7 spaces inlined instead of 1tab==8 spaces. It is at the level of the whole `()`.

Reply:
This indent passes checkpatch, so the coding style should be OK.